manifests/default.nix

```diff
@@ -7,9 +7,9 @@
   callPackage,
 }: let
   mkPursDerivation = callPackage ./build-support/mkPursDerivation.nix {};
-  mkSpagoDerivation = callPackage ./build-support/mkSpagoDerivation.nix {};
-  mkPursTidyDerivation = callPackage ./build-support/mkPursTidyDerivation.nix {};
-  mkPursBackendEsDerivation = callPackage ./build-support/mkPursBackendEsDerivation.nix {};
+  mkSpagoDerivation = callPackage (callPackage ./build-support/mkSpagoDerivation.nix {}) {};
+  mkPursTidyDerivation = callPackage (callPackage ./build-support/mkPursTidyDerivation.nix {}) {};
+  mkPursBackendEsDerivation = callPackage (callPackage ./build-support/mkPursBackendEsDerivation.nix {}) {};
```

Comment:
The nested `callPackage` here feels a bit weird to me. Is this necessary? Could we change the order of arguments so we are only calling `callPackage` once?

Reply:
Yes, it looks a bit weird. 

I'm not aware of `callPackage` that can process 2 arguments. 

In fact, 

```nix
callPackage ./build-support/mkSpagoDerivation.nix {}
```

is a shorter alternative to

```nix
import ./build-support/mkSpagoDerivation.nix { inherit lib; }
```

If necessary, I can rewrite in the second way.

Reply:
`lib` is necessary only for `meta`, so I made `meta` argument a function of `lib`.